src/EditorFeatures/Core.Wpf/Lightup/LightupHelpers.cs

```diff
@@ -352,6 +408,91 @@ public static Action<T> CreateActionAccessor<T>(Type? type, string methodName)
         return expression.Compile();
     }
 
+    /// <summary>
+    /// Generates a compiled accessor method for a property which cannot be bound at compile time.
+    /// </summary>
+    /// <typeparam name="T">The compile-time type representing the instance on which the property is defined. This
+    /// may be a superclass of the actual type on which the property is declared if the declaring type is not
+    /// available at compile time.</typeparam>
+    /// <typeparam name="TArg">The compile-time type representing the type of the first argument. This
+    /// may be a superclass of the actual type of the argument if the declared type is not available at compile
+    /// time.</typeparam>
+    /// <typeparam name="TResult">The compile-type type representing the result of the property. This may be a
+    /// superclass of the actual type of the property if the property type is not available at compile
+    /// time.</typeparam>
+    /// <param name="type">The runtime time on which the property is defined. If this value is null, the runtime
+    /// time is assumed to not exist, and a fallback accessor returning <paramref name="defaultValue"/> will be
+    /// generated.</param>
+    /// <param name="methodName">The name of the method to access.</param>
+    /// <param name="defaultValue">The value to return if the method is not available at runtime.</param>
+    /// <returns>An accessor method to access the specified runtime property.</returns>
+    public static Func<T, TArg0, TArg1, TResult> CreateFunctionAccessor<T, TArg0, TArg1, TResult>(Type? type, string methodName, Type? arg0Type, Type? arg1Type, TResult defaultValue)
+    {
+        if (methodName is null)
+        {
+            throw new ArgumentNullException(nameof(methodName));
```

Comment:
these exceptions are terrifying.  who catches them?  does this tear down VS?

Reply:
this is a work in progress, I'll address it

Reply:
We need to manually test these helpers after all the comments are addressed.
(We used to break the public preview one time)